pkgs/development/python-modules/oterm/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, lib, buildPythonPackage, fetchFromGitHub
+, poetry-core, textual, typer, python-dotenv, httpx
+, aiosql, aiosqlite, pyperclip, packaging
+, ruff, black, pytest, pytest-asyncio, textual-dev, aiohttp }:
+
+buildPythonPackage rec {
```

Comment:
I think `oterm` is an application, not a library.
Could you please use `buildPythonApplication` and move to `pkgs/by-name/ot/oterm` instead?